extension/src/test/e2e/extension.test.ts

```diff
@@ -230,16 +230,12 @@ describe('Source Control View', function () {
           const tooltip = await findDecorationTooltip(treeItem)
           expect(tooltip).toBeTruthy()
         }
-        return expectedScmItemLabels.length === dvcTreeItemLabels.length
+        return dvcTreeItemLabels.length > 0
```

Comment:
See [this](https://github.com/iterative/vscode-dvc/pull/4103#discussion_r1228898516)